FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c

```diff
@@ -500,57 +500,6 @@ void test_coverage_prvYieldCore_runstate_eq_yielding( void )
     TEST_ASSERT_EQUAL( 1, task.xTaskRunState );                 /* nothing has changed */
 }
 
-/**
- * @brief vTaskDelete - scheduler not running.
- *
- * This test ensures that if vTaskDelete is called and the scheuler is
- * not running, the core is not yielded, but it is removed from the
- * stateList, the eventList and inserted in the taskwaitingtermination
- * list, the uxdeletedtaskwaiting for cleanup is increased and the
- * uxtasknumber is increased
- *
- * <b>Coverage</b>
- * @code{c}
- * if( ( xSchedulerRunning != pdFALSE ) &&
- *     ( taskTASK_IS_RUNNING( pxTCB ) == pdTRUE ) )
- * ...
- * @endcode
- * ( xSchedulerRunning != pdFALSE ) is false.
- */
-void test_coverage_vTaskDelete_scheduler_not_running( void )
```

Comment:
Why don't we have this scenario? It's still necessary to have an expectation when users call vTaskDelete if scheduler's not running.

Reply:
The code to be covered by this test is removed in the implementation. This is also checked in the coverage report.